src/CattyTests/Bricks/SetLookByIndexBrickTests.swift

```diff
@@ -0,0 +1,123 @@
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+
+import XCTest
+
+@testable import Pocket_Code
+
+final class SetLookByIndexBrickTest: XCTestCase {
+
+    var project: Project!
+    var spriteObject: SpriteObject!
+    var spriteNode: CBSpriteNode!
+    var script: Script!
+    var brick: SetLookByIndexBrick!
+    var formulaInterpreter: FormulaInterpreterProtocol!
+
+    var look: Look!
+    var look2: Look!
+    var look3: Look!
+
+    override func setUp() {
+        project = ProjectManager.createProject(name: "setLookByIndexTest", projectId: "1")
+        spriteObject = SpriteObject()
+        spriteObject.scene = project.scene
+        spriteObject.name = "SpriteObjectName"
+
+        spriteNode = CBSpriteNode(spriteObject: spriteObject)
+        spriteObject.spriteNode = spriteNode
+
+        script = Script()
+        script.object = spriteObject
+
+        formulaInterpreter = FormulaManager(stageSize: Util.screenSize(true), landscapeMode: false)
+
+        let bundle = Bundle(for: type(of: self))
+        let filePath = bundle.path(forResource: "test.png", ofType: nil)
+        let imageData = UIImage(contentsOfFile: filePath!)!.pngData()
+
+        do {
+            look = Look(name: "test", filePath: "test.png")
+            try imageData?.write(to: URL(fileURLWithPath: spriteObject.scene.imagesPath()! + "/test.png"))
+            look2 = Look(name: "test2", filePath: "test2.png")
+            try imageData?.write(to: URL(fileURLWithPath: spriteObject.scene.imagesPath()! + "/test2.png"))
+            look3 = Look(name: "test3", filePath: "test3.png")
+            try imageData?.write(to: URL(fileURLWithPath: spriteObject.scene.imagesPath()! + "/test3.png"))
+        } catch {
+            XCTFail("Error when writing image data")
+        }
+
+        brick = SetLookByIndexBrick()
+        brick.script = script
+
+        spriteObject.lookList.add(look!)
+        spriteObject.lookList.add(look2!)
+        spriteObject.lookList.add(look3!)
+        spriteObject.spriteNode.currentLook = look
+        spriteObject.spriteNode.currentUIImageLook = UIImage(contentsOfFile: filePath!)
+    }
+
+    func testSetLookByValidIndex() {
+        brick.lookIndex = Formula(integer: 3)
+
+        var action = brick.actionBlock(formulaInterpreter)
+        action()
+        XCTAssertEqual(spriteNode.currentLook, look3, "setLookByIndexBrick not correct")
```

Comment:
What do you mean with "setLookByIndexBrick not correct"? I think this is rather confusing, therefore I would suggest, to either remove the String or to rephrase it. And please do so for all the other XCTAssertEquals in this file.